docs/configuring_tor.md

```diff
@@ -86,7 +86,7 @@ Tor:
       --tor.privatekeypath=                                   The path to the private key of the onion service being created
 ```
 
-There are a couple things here, so let's dissect them. The `--tor.active` flag
+There are a couple of things here, so let's dissect them. The `--tor.active` flag
```

Comment:
just eye-balling this but looks like the wrapping is now off?

Reply:
Fixed.